README.md

```diff
@@ -444,7 +444,8 @@ If the `enabled` flag is `true`, it will be started automatically.
 
 All host containers will have the `apiclient` binary available at `/usr/local/bin/apiclient` so they're able to [interact with the API](#using-the-api-client).
 
-In addition, all host containers come with persistent storage at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME` that is persisted across reboots and container start/stop cycles.
+In addition, all host containers come with persistent storage that survives reboots and container start/stop cycles.
+It's available at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME` and (since Bottlerocket v1.0.8) `/.bottlerocket/host-containers/current`.
```

Comment:
I don't like putting a specific version number here - we may need to cut a 1.0.8 release for an emergency security or operational fix, and it'd be easy to overlook references like this.

Reply:
I suggested this because I didn't like having a documented feature that wasn't available yet, and we've used similar terminology elsewhere. We can update this if there's an emergency. I'd you don't like this, what would you suggest?

Reply:
I don't love it, but it looks like it's consistent with our documentation for the `user-data` feature in v1.0.6, so I'm OK with keeping it.